packages/eslint-plugin/tests/lib/rules/no-use-before-define.js

```diff
@@ -177,8 +177,8 @@ var bar;
     // "typedefs" option
     {
       code: `
-var x: Foo = 2;
 type Foo = string | number
+var x: Foo = 2;
```

Comment:
were you meant to move this?
the test was supposed to be valid before due to the `typedefs` option

Reply:
you are right :)

Reply:
right now types are completely ignored :)